package.json

```diff
@@ -70,6 +70,9 @@
   "peerDependencies": {},
   "dependencies": {
     "@types/uuid": "^3.4.5",
-    "azure-maps-control": "^2.0.14"
+    "azure-maps-control": "^2.0.14",
+    "guid-typescript": "^1.0.9",
+    "mapbox-gl": "^1.4.1",
```

Comment:
Do we need this mapbox-gl library?

Reply:
Yep, without that we will have pretty bad looking warning that we don't have styles for map-box loaded on page 

```
// Styles section
import 'mapbox-gl/src/css/mapbox-gl.css'
```